pkgs/sketch_pad/lib/widgets.dart

```diff
@@ -107,6 +108,9 @@ class RunButton extends StatelessWidget {
               RoundedRectangleBorder(borderRadius: BorderRadius.circular(4.0))),
           backgroundColor: MaterialStateProperty.resolveWith(
             (states) {
+              if (disabled) {
```

Comment:
If _may_ be enough to pass in `null` for `onPressed:` below; its at least worth trying.

Reply:
Already, I will submit another PR with this suggestion.